docs/alerts.rst

```diff
@@ -0,0 +1,242 @@
+Alerts
+======
+
+Alerts can be used to notify you about important App Control-monitored activities, such as the
+appearance or spread of risky files on your endpoints. The Carbon Black Cloud Python SDK provides
+all of the functionalities you might need to use them efficiently.
+You can use all of the operations shown in the API such as retrieving, filtering, dismissing, creating and updating.
+The full list of operations and attributes that are available and can be used, can be found in the :py:mod:`BaseAlert() <cbc_sdk.platform.alerts.BaseAlert>` class.
+
+For more information see
+`the developer documentation <https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/alerts-api/>`_
+
+Retrieving of Alerts
+--------------------
+
+In the example below we can retrieve alerts with the minimum severity of ``7``.
```

Comment:
Consider changing to "With the example below, you can retrieve alerts with the minimum severity of ``7``.